test/compatibility/src/main/resources/logback-test.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Copyright (c) 2017 Dell Inc., or its subsidiaries.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+-->
+<configuration scan="true">
```

Comment:
We have no code in `src/main` , so I'm wondering if we need this logback configuration file. Also, why is it calle `logback-test.xml`?